.github/workflows/release-orchestrate-overall.yml

```diff
@@ -7,82 +7,127 @@ on:
         description: 'The version that is going to be released. Should be in format 7.y.z'
         required: true
         default: ''
-      phases:
-        description: 'List of comma-separated phases to perform; default should be "1,2,3,4,5,6,7"; omit phases that have successfully run'
```

Comment:
if you remove this easy-to-use trigger mechanism, we need a replacement. Not sure if a list of projects is simpler than phases since we'd have to know which projects have prereqs/postreqs  (eg., plugin reg depends on success of machine exec and theia, but having them combined as "phase 2" is easier to type. :D